snapshots/education/2023-07-17/education_barro_lee_projections.py

```diff
@@ -0,0 +1,65 @@
+"""Script to create a snapshot of dataset 'Human Capital in the Long Run (Lee and Lee, 2016)'."""
```

Comment:
maybe add a short text explaining what this script does.

E.g. access & download several CSVs, concatenate thee and save them as one snapshot